src/components/accession2/Accession2Edit.tsx

```diff
@@ -0,0 +1,181 @@
+  const goToAccession2View = () => {
+    if (accession) {
+      history.push(APP_PATHS.ACCESSIONS2_ITEM.replace(':accessionId', accession.id.toString()));
+    }
+  };
+
+  const saveAccession = async () => {
+    if (record) {
+      const response = await updateAccession2(record);
+      if (response.requestSucceeded && accession) {
+        history.push(APP_PATHS.ACCESSIONS2_ITEM.replace(':accessionId', accession.id.toString()));
+      }
+    }
+  };
+
+  return (
+    <Grid container>
+      <Grid container item xs={12} padding={4} paddingBottom={'92px'}>
+        <Grid item xs={12}>
+          <h1>{strings.EDIT_ACCESSION}</h1>
+        </Grid>
+        {accession ? (
+          <Grid container xs={12} spacing={2}>
+            <Grid item xs={mainStructureSize}>
+              <Textfield
+                id='accessionNumber'
+                type='text'
+                label={strings.ACCESSION_ID}
+                value={record?.accessionNumber}
+                onChange={onChange}
+              />
+            </Grid>
+
+            <Grid item xs={mainStructureSize}>
+              <Textfield
+                id='speciesCommonName'
+                type='text'
+                label={strings.SPECIES}
+                value={record?.speciesCommonName}
+                onChange={onChange}
+              />
+            </Grid>
+
+            <Grid item xs={mainStructureSize}>
+              <Textfield
+                id='collectedDate'
+                type='text'
+                label={strings.COLLECTING_DATE_REQUIRED}
+                value={record?.collectedDate}
+                onChange={onChange}
+              />
+            </Grid>
+
+            <Grid item xs={mainStructureSize}>
+              <Textfield
+                id='collector'
+                type='text'
+                label={strings.COLLECTOR}
+                value={record?.collectors ? record?.collectors[0] : ''}
+                onChange={onChange}
+              />
+            </Grid>
+
+            <Grid item xs={mainStructureSize}>
+              <Textfield
+                id='collectionSiteName'
+                type='text'
+                label={strings.COLLECTING_SITE}
+                value={record?.collectionSiteName}
+                onChange={onChange}
+              />
+            </Grid>
+
+            <Grid item xs={mainStructureSize}>
+              <Textfield
+                id='collectionSiteLandowner'
+                type='text'
+                label={strings.LANDOWNER}
+                value={record?.collectionSiteLandowner}
+                onChange={onChange}
+              />
+            </Grid>
+            <Grid item xs={mainStructureSize}>
+              <Textfield
+                id='collectionSiteCity'
+                type='text'
+                label={strings.CITY}
+                value={record?.collectionSiteCity}
+                onChange={onChange}
+              />
+            </Grid>
+            <Grid item xs={6}>
```

Comment:
looks too small on mobile, should be 12 maybe?

Reply:
Yes, I will update it